pyproject.toml

```diff
@@ -58,7 +58,7 @@ aioshutil = "^1.3"
 aiofiles = "^23.2.1"
 arabic-reshaper = "^3.0.0"
 pyhyphen = "^4.0.3"
-pydensecrf = { git = "https://github.com/kodalli/pydensecrf.git" }
+pydensecrf = { git = "https://github.com/lucasb-eyer/pydensecrf.git" }

```

Comment:
I'm not a poetry user.
The lock file is updated with `poetry lock --no-update` and `unix2dos` . IDK if this is the supposed way to do it 🤔